docs/usage.md

```diff
@@ -68,7 +100,18 @@ The widgets within the solution pack's dashboard, such as the Record Summary Car
 The "Global Visibility" playbook collection plays a pivotal role in the FortiSOAR Global Visibility Solution Pack by aggregating, centralizing, and transforming data from individual sites into a unified dataset. This dataset is harnessed by the solution pack's widgets to provide users with insightful and consolidated visualizations of alerts, incidents, tasks, and other critical metrics across multiple FortiSOAR instances. The "Global Visibility" module acts as the foundation for enhancing decision-making and security operations efficiency.
 
 
+### Delete Stale Data Playbook
+This playbook syncs records with the sites configured in Remote FortiSOAR Connector and deletes records from Global Visibility Data module whose respective Site's configuration is either deleted or health check is unavailable.
+
+![Global Visibility Scenario](./res/global-visibility-data-02.png)
+
+### Refresh Dashboard Data Playbook
+This playbook syncs records with the sites configured in Remote FortiSOAR Connector and creates/updates records for sites based on Remote FortiSOAR connector configured.
```

Comment:
This playbook syncs records with the sites configured in Remote FortiSOAR Connector and creates or updates records for those sites.